tests/test_parse_load_args.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import unittest
+import uuid
+from typing import List, Optional
+
+import mkosi
+import mkosi.backend as backend
+from mkosi.backend import MkosiArgs, MkosiException, Distribution
+
+
+def load_args_wrapper(argv: Optional[List[str]] = None) -> MkosiArgs:
```

Comment:
This is a very long name for a local function. Since we use this a lot and only in this file, we can get away with just calling it parse() I think.